main.js

```diff
@@ -0,0 +1,44 @@
+$("#submit-post").click(function () { 
+   
+  var author = $("#post-author").val();
+  var text = $("#post-text").val();
+  
+
+  var remove = '<button class="remove">Remove</button>';
+  var comments = '<button class="comments">Comments</button>';
+  var comment_section = '<div class="col-8 comment-section"><div id="new-comment"></div><form><div class="form-group">  <input type="text" class="form-control" id="comment-text"  placeholder="Comment text"</div><div class="form-group">  <input type="text" class="form-control" id="comment-author"  placeholder="Your Name"</div><button type="button" class="btn btn-primary" id="submit-comment">  Submit Comment</button></form></div>';  
+  
+  $("#new-post").append('<p class="post">' + remove + comments + text + " - Posted By: " + author  + '</p>' + comment_section );
+
+  $("form").trigger("reset");
+});
+
+//removes post
+$(document).on('click','.remove',  function() { 
+  $(this).closest('div').find('.comment-section').remove();
+  $(this).parent().remove(); 
+  
+});
+
+//opens and closes comment section on post
+$(document).on('click', '.comments', function () {  
+  $(this).closest('div').find('.comment-section').toggle();   
+});
+
+//adds comment
+$(document).on('click', '#submit-comment', function() {
+  
+  var commentAuthor = $("#comment-author").val();
+  var commentText = $("#comment-text").val();
```

Comment:
Chaining several jQuery methods like this works, but makes your code less robust (if the DOM changes, your code could break). I'd recommend considering other options here in the future.